simphony/material_relations/material_relation.py

```diff
@@ -0,0 +1,118 @@
+from simphony.material_relations.abc_material_relation import (
+    ABCMaterialRelation)
+import simphony.core.data_container as dc
+
+
+class MaterialRelation(ABCMaterialRelation):
+    """ MaterialRelation provides a general interface for describing the
+    (physics/chemistry) relations between different materials
+
+    Attributes
+    ----------
+    name : str
+        name of the material-relation
+
+    description: str
+        user-defined description of the material-relation
+
+    parameters: DataContainer
+        the required parameters
+
+    supported_parameters: list of CUBA
+        CUBA values required/allowed for the parameters
+
+    materials: list of uids
+        materials where this relation applies
+
+    num_materials: list of int
```

Comment:
I think 'num_materials' should be removed from both the __init__ parameter list as well as from this class attributes.  The number of materials can be derived from the 'kind'.